spec/features/questions/show_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+require 'pry'
+RSpec.describe "display all questions" do
+  it "has all expected elements" do
+    a = RatingQuestion.create!(title: "something")
```

Comment:
Set this question up using a `let` block in your test.

Reply:
ok